Dockerfile

```diff
@@ -155,10 +160,14 @@ RUN npm i -g node-sass@4.10.0 --unsafe-perm=true
 RUN npm i -g optipng-bin@5.0.0 --unsafe-perm=true
 RUN npm i -g jpegtran-bin@4.0.0 --unsafe-perm=true
 RUN npm i -g gifsicle@4.0.1 --unsafe-perm=true
+RUN npm i -g lighthouse@5.6.0 --unsafe-perm=true
```

Comment:
@mattsbanner You don't need to install lighthouse at a specific version number we do this with the other packages just for speed as we try and match what the build script wants to install so those deps already exist at the right versions.